src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/summary/MethodologySummaryPage.tsx

```diff
@@ -3,84 +3,70 @@ import {
 import WarningMessage from '@common/components/WarningMessage';
-import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
+import { useMethodologyContext } from '@admin/pages/methodology/contexts/MethodologyContext';
 import React from 'react';
-import { generatePath, RouteComponentProps } from 'react-router';
+import { generatePath } from 'react-router';
 
-const MethodologySummaryPage = ({
-  match,
-}: RouteComponentProps<MethodologyRouteParams>) => {
-  const { methodologyId } = match.params;
-
-  const { value: currentMethodology, isLoading } = useAsyncHandledRetry(() =>
-    methodologyService.getMethodology(methodologyId),
-  );
+const MethodologySummaryPage = () => {
+  const { methodologyId, methodology } = useMethodologyContext();
 
   return (
     <>
       <h2>Methodology summary</h2>
 
-      <LoadingSpinner loading={isLoading}>
-        {currentMethodology ? (
-          <>
-            <SummaryList>
-              <SummaryListItem term="Title">
-                {currentMethodology.title}
-              </SummaryListItem>
-              <SummaryListItem term="Published on">
-                {currentMethodology.published ? (
-                  <FormattedDate>{currentMethodology.published}</FormattedDate>
-                ) : (
-                  'Not yet published'
-                )}
-              </SummaryListItem>
-              <SummaryListItem term="Owning publication">
-                {currentMethodology.owningPublication.title}
-              </SummaryListItem>
-              {currentMethodology.otherPublications &&
-                currentMethodology.otherPublications.length > 0 && (
-                  <SummaryListItem term="Other publications">
-                    <ul className="govuk-!-margin-top-0">
-                      {currentMethodology.otherPublications?.map(
-                        publication => (
-                          <li
-                            key={publication.id}
-                            data-testid="other-publication-item"
-                          >
-                            {publication.title}
-                          </li>
-                        ),
-                      )}
-                    </ul>
-                  </SummaryListItem>
-                )}
-            </SummaryList>
+      {methodology ? (
+        <>
+          <SummaryList>
+            <SummaryListItem term="Title">{methodology.title}</SummaryListItem>
+            <SummaryListItem term="Published on">
+              {methodology.published ? (
+                <FormattedDate>{methodology.published}</FormattedDate>
+              ) : (
+                'Not yet published'
+              )}
+            </SummaryListItem>
+            <SummaryListItem term="Owning publication">
+              {methodology.owningPublication.title}
+            </SummaryListItem>
+            {methodology.otherPublications &&
+              methodology.otherPublications.length > 0 && (
```

Comment:
Can make this a little more succinct with: `methodology.otherPublications?.length > 0`

Reply:
Same as above.